[2023-12-30T13:03:08.661+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.sense_json_file scheduled__2023-12-28T00:00:00+00:00 [queued]>
[2023-12-30T13:03:08.743+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.sense_json_file scheduled__2023-12-28T00:00:00+00:00 [queued]>
[2023-12-30T13:03:08.747+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-30T13:03:08.954+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): sense_json_file> on 2023-12-28 00:00:00+00:00
[2023-12-30T13:03:08.971+0000] {standard_task_runner.py:57} INFO - Started process 322 to run task
[2023-12-30T13:03:08.991+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'sense_json_file', 'scheduled__2023-12-28T00:00:00+00:00', '--job-id', '174', '--raw', '--subdir', 'DAGS_FOLDER/dag0001.py', '--cfg-path', '/tmp/tmps2sxfr_q']
[2023-12-30T13:03:09.010+0000] {standard_task_runner.py:85} INFO - Job 174: Subtask sense_json_file
[2023-12-30T13:03:09.302+0000] {task_command.py:416} INFO - Running <TaskInstance: my_dag.sense_json_file scheduled__2023-12-28T00:00:00+00:00 [running]> on host e1038f319166
[2023-12-30T13:03:09.721+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Javad' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='sense_json_file' AIRFLOW_CTX_EXECUTION_DATE='2023-12-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-28T00:00:00+00:00'
[2023-12-30T13:03:09.729+0000] {logging_mixin.py:154} INFO - Sensing JSON file
[2023-12-30T13:03:09.730+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-30T13:03:09.779+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=sense_json_file, execution_date=20231228T000000, start_date=20231230T130308, end_date=20231230T130309
[2023-12-30T13:03:09.907+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-30T13:03:10.151+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
